wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
Yes please.

(Also technically, your version number bump commit is *before* the code commit...so anyone looking at a release of pypi 0.1.5 would expect this change to *not* be in there, and potentially be confused as to why it was.)